README.tpl

```diff
@@ -1,42 +0,0 @@
-**_Please be aware that version `v0.4.0` of rusty-cheddar will be based around the more general
```

Comment:
I guess you want the template (tpl) under version control.  Ideally you wouldn't check the result (md) in at all, but GitHub kinda requires it.